bin/start_do.sh

```diff
@@ -354,4 +357,4 @@ echo "Main Configuration: pulumi config -C ${script_dir}/../pulumi/python/config
 echo "Bank of Sirius (Example Application) Configuration: pulumi config -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius"
 echo "K8 Loadbalancer IP: kubectl get services --namespace nginx-ingress"
 echo " "
-echo "Please see the documentation in the github repository for more information"
+echo "Please see the documentation in the github repository for more information"
```

Comment:
I think this message is great!  Quick formatting question: I've been using `cat` with a heredoc to do these kinds of messages in my project.  Are multiple `echo` lines preferred?

Reply:
I've done both, to be honest. If we were keeping these bash scripts around I would probably rework it to use a function that I then just pass the messages to so I can do things like tee to a log, put in timestamps, etc.